### PR TITLE
Add updated sponsor logos in powered section

### PR DESCRIPTION
**Pull request by ribeiropdiogo**
Added updated logos in powered by section

### CODE REVIEW

**Comment by machadovilaca** [comment]
Please change the description to English and capitalize PR title like the others

File: _assets/stylesheets/home/_header.sass [hunk 1]
```diff
@@ -53,3 +53,7 @@
     #registerbutton
       margin-top: 30px
       font-size: 20px
+
+.vertical-middle
+  display: flex
+  align-items: center
```
Comment (machadovilaca) [inline _assets/stylesheets/home/_header.sass]: Add empty line at the end of file

**Review by machadovilaca — DISMISSED** [review]
Personally I think the 'powered by' text should be lowercase and the images seem way too big

**Comment by miguelbrandao** [comment]
> Personally I think the 'powered by' text should be lowercase

I'm not sure if it looks better
image

I'm conflicted :sweat_smile:

**Comment by machadovilaca** [comment]
The 2 parts of the comment are correlated, only the text at lowercase looks bad because the images are still to big

**Comment by ribeiropdiogo** [comment]
I'm confused, powered by, uppercase or lowercase? The images are already smaller.

**Comment by miguelbrandao** [comment]
I think the Pfizer logo is a nice size, but the other one looks small in comparison. Can you try to scale it to match the Pfizer one? And maybe add a bit of spacing between them, but not too much :sweat_smile: 

And let's give a shot to the lowercase "powered by"

**Comment by ribeiropdiogo** [comment]
Ok then. The spacing is already there. I've put 5px left and right, but I will add more.

**Review by miguelbrandao — APPROVED** [review]

**Comment by machadovilaca** [comment]
Looking better, right? :D

**Pull request merged**